.codegen/service.go.tmpl

```diff
@@ -145,7 +145,13 @@ func new{{.PascalName}}() *cobra.Command {
 	{{- end}}
 	{{end}}
 
-	{{- $excludeFromPrompts := list "workspace get-status" -}}
+	{{- $excludeFromPrompts := list
+		"workspace get-status"
+		"provider-exchanges get"
```

Comment:
temporarily disabled until resolved on SDK side (need to store Request reference in NamedIdMap so the correct request type is used in request later)